docker-compose.yml

```diff
@@ -32,7 +32,7 @@ services:
     deploy:
       resources:
         limits:
-          memory: 20M
+          memory: 120M
```

Comment:
The memory sizes look good, but should we add a recommended cluster size with X GBs memory to run the docker compose in our getting started documentation?